rewards/snapshot/chain_snapshot.py

```diff
@@ -9,6 +9,8 @@
 
 console = Console()
 
+skip = ["0xb6bd5ae3d5f78a6bb04bbb031e24fa9c2bbd090d"]
```

Comment:
Let's rename this to something that has more meaning than `skip`.  `invalid_vaults` maybe?